test/integration/amateur_test.rb

```diff
@@ -428,6 +428,7 @@ def create_name(obs, text_name)
         form.change('name', "#{text_name} #{author}")
         form.change('reason_2_check', true)
         form.change('reason_2_notes', reason)
+        form.select('vote_value', /call it that/i)
```

Comment:
Because the default "vote" when creating an observation is now blank, we need to select an option when creating an observation.